### PR TITLE
Add @sudo_overflow

### PR DESCRIPTION
**Pull request by CyrisXD**

### CODE REVIEW

File: data/users.json [hunk 1]
```diff
@@ -257,6 +257,24 @@
       "Docker"
     ]
   },
+  {
+    "username": "Sudo_overflow",
+    "stack": [
+      "HTML",
```
Comment (0xtelmo) [inline data/users.json]:
```
HTML5
```

File: data/users.json [hunk 2]
```diff
@@ -257,6 +257,24 @@
...
+      "CSS",
```
Comment (0xtelmo) [inline data/users.json]:
```
CSS3
```

**Comment by 0xtelmo** [comment]
Just to match the icons over at SimpleIcons 🙂

**Comment by CyrisXD** [comment]
That should be updated, pushed changes to my Master.

**Review by 0xtelmo — APPROVED** [review]

**Pull request merged**